index.html

```diff
@@ -0,0 +1,93 @@
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <link rel="stylesheet" media="(max-width:320px)" href="style_0_320.css">
+        <link rel="stylesheet" media="(min-width:321px) and (max-width:640px)" href="style_320_640.css">
+        <link rel="stylesheet" media="(min-width:641px) and (max-width:980px)" href="style_640_980.css">
+        <link rel="stylesheet" media="(min-width:981px)" href="style_980_1024.css">
+        <link rel="stylesheet" media="(max-width:640px)" href="menu_hidden.css">
+        <link rel="stylesheet" media="(min-width:641px)" href="menu.css">
+        <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css" integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    </head>
+    <body>
+        <nav>
+            <span>Navigation</span>
+            <div class="menu-wrapper">
+                <i class="fas fa-bars"></i>
+                <ul>
+                    <li>
+                        <a href="#">PSD WEBSITES</a>
+                    </li>
+                    <li>
+                        <a href="#">UI KITS</a>
+                    </li>
+                    <li>
+                        <a href="#">PREMIUM PSD FILES</a>
+                    </li>
+                    <li>
+                        <a href="#">ABOUT</a>
+                    </li>
+                    <li>
+                        <a href="#">BLOG</a>
+                    </li>
+                </ul>
+            </div>
+        </nav>
+        <a href="http://blazrobar.com/" class="banner">
+            <span class="banner-content">
+                <span class="banner-info">
+                    Awesome Free PSD<br>
+                    templates & Resources
+                </span>
+                <span class="banner-link">
+                    www.BlazRobar.com
+                </span>
+                <i class="fas fa-angle-right banner-ico"></i>
+            </span>
+        </a>
+        <div class="text-box">
+            <div class="text-box-content">
+                <article>
+                    I'm like the Robin Hood of PSD files.
+                </article>
+            </div>
+            <a href="http://blazrobar.com/" class="text-box-link">
+                <i class="fas fa-angle-right"></i>
+            </a>
+        </div>
+        <div class="galery">
+            <a href="http://blazrobar.com/" class="galery-link galery-item">
+                <img src="Galery-link.jpg">
+                <i class="fas fa-angle-right"></i>
+            </a>
+            <img src="Bridge.jpg" class="galery-item">
+            <img src="Woman.jpg" class="galery-item">
+            <div class="galery-breaker">
+            </div>
+        </div>
+        <div class="main-content">
```

Comment:
If it's a main-content block, maybe <main> could be more useful instead of <div>?